models/Site.php

```diff
@@ -470,4 +473,32 @@ public function getCreationDate()
     {
         return $this->creationDate;
     }
+
+    /**
+     * @return void
+     */
+    public function save()
+    {
+        $preSaveEvent = new SiteEvent($this);
+        $this->dispatchEvent($preSaveEvent, SiteEvents::PRE_SAVE);
+
+        $this->__call(__FUNCTION__, []);
+
+        $postSaveEvent = new SiteEvent($this);
+        $this->dispatchEvent($postSaveEvent, SiteEvents::POST_SAVE);
+    }
+
+    /**
+     * @return void
+     */
+    public function delete()
+    {
+        $preDeleteEvent = new SiteEvent($this);
+        $this->dispatchEvent($preDeleteEvent, SiteEvents::PRE_DELETE);
+
+        $this->__call(__FUNCTION__, []);
```

Comment:
```suggestion
        $this->getDao()->delete();
```
This is more accessible 😉

Reply:
done thx :+1: